tests/cluster/schema_manager/test_schema_manager.py

```diff
@@ -52,20 +56,22 @@ def test_get_all_schemas(self):
         self.assertDictEqual(schemas_dict, expected_schemas)
 
     def test_get_all_spark_sql_schemas(self):
-        Configurator().add_resource_path(extras)
-
         schemas_dict = SchemaManager().get_all_spark_sql_schemas()
 
         test_table_string = (
-            "a int, b int, c string",
```

Comment:
reformatted for readability, also, some commas were outside the quotation marks which made the whole thing a tuple